skymp5-server/cpp/lib_espm/Loader.cpp

```diff
@@ -0,0 +1,16 @@
+#include "Loader.h"
+
+#include "impl/AllocatedBuffer.h"
+#include "impl/MappedBuffer.h"
+
+namespace espm {
+
+std::unique_ptr<impl::IBuffer> Loader::MakeBuffer(
+  const fs::path& filePath) const
+{
+  // return std::unique_ptr<impl::IBuffer>{new
+  // impl::AllocatedBuffer(filePath)};
+  return std::unique_ptr<impl::IBuffer>{ new impl::MappedBuffer(filePath) };
```

Comment:
Let's add a constructor parameter to `Loader` and for now, use `MappedBuffer` for the server and unit tests.